analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AppSettings/DatabasePasswordsShouldBeSecure/UnexpectedContent/appsettings.json

```diff
@@ -0,0 +1,5 @@
+﻿{
```

Comment:
could you also add tests with an empty file and with a JSON with an empty array `[]` (just thinking of other edge cases)